docs/internals/tech_overview.rst

```diff
@@ -58,69 +58,6 @@ Execution guide
 - Then, depending on the ``task`` option, it might start ``engine.train``, ``engine.forward`` etc.
   (:py:func:`Engine.Engine.train` or :py:func:`TFEngine.Engine.train`), :ref:`tech_engine_train`.
 
-
-.. _tech_net_construct:
-
-Network structure construction
-------------------------------
-
-The network structure which defines the model topology is defined by the config ``network`` option,
-which is a dict, where each entry is a layer specification, which itself is a dict containing
-the kwargs for the specific layer class. E.g.:
-
-.. code-block:: python
-
-    network = {
-        "fw1": {"class": "linear", "activation": "relu", "dropout": 0.1, "n_out": 500},
-        "fw2": {"class": "linear", "activation": "relu", "dropout": 0.1, "n_out": 500, "from": ["fw1"]},
-        "output": {"class": "softmax", "loss": "ce", "from": ["fw2"]}
-    }
-
-The ``"class"`` key will get extracted from the layer arguments and the specific layer class will be used.
```

Comment:
Where is this text now? This is important for understanding how the network construction works.

Reply:
I did not remove any text at all, this is part of the new files. This also holds for the next 4 comments. The tech overview should be changed, as the Theano information is more confusing than helping (it is deprecated anyways). Information on how to use RETURNN should all be in "basic_usage", and the information here should be organized in a different way. I also do not understand the name "technological overview", because this sounds like a description of available features of RETURNN, not like a list of internal classes. I will continue working on this when i have the time.

Reply:
But this tech overview was mostly about TF. It explains that it extracts the `class` key, calls `construct_from_dict`, and all this stuff. I don't see that this is part of the new files. Where exactly? In my opinion, this is some of the most important part of the documentation.

Reply:
Ah sorry, now I see it.